test/simulation.jl

```diff
@@ -942,6 +942,83 @@ end
     end
 end
 
+@testset "Monte Carlo anisotropic barostat" begin
+    # See http://www.sklogwiki.org/SklogWiki/index.php/Argon for parameters
+    n_atoms = 25
+    n_steps = 1_000_000
+    atom_mass = 39.947u"u"
+    boundary = CubicBoundary(8.0u"nm")
+    temp = 288.15u"K"
+    dt = 0.0005u"ps"
+    friction = 1.0u"ps^-1"
+    lang = Langevin(dt=dt, temperature=temp, friction=friction)
+    atoms = fill(Atom(mass=atom_mass, σ=0.3345u"nm", ϵ=1.0451u"kJ * mol^-1"), n_atoms)
+    coords = place_atoms(n_atoms, boundary; min_dist=1.0u"nm")
+    n_log_steps = 500
+    
+    box_volume_wrapper(sys, neighbors; kwargs...) = box_volume(sys.boundary)
+    VolumeLogger(n_steps) = GeneralObservableLogger(box_volume_wrapper, typeof(1.0u"nm^3"), n_steps)
+    
+    baro_f(pressure) = MonteCarloAnisotropicBarostat(pressure, temp, boundary)
+    lang_f(barostat) = Langevin(dt=dt, temperature=temp, friction=friction, coupling=barostat)
+    vand_f(barostat) = VelocityVerlet(dt=dt, coupling=(AndersenThermostat(temp, 1.0u"ps"), barostat))
+    
+    pressure_test_set = (
+        SVector(1.0u"bar", 1.0u"bar", 1.0u"bar"), # XYZ-axes coupled with the same pressure value
+        SVector(1.5u"bar", 0.5u"bar", 1.0u"bar"), # XYZ-axes coupled with different pressure values
+        SVector(1.0u"bar", nothing, nothing), # Only X-axis coupled
```

Comment:
Just test one of the x/y/z single couplings, for speed.

Reply:
I removed `only x-axis` and `only z-axis`.